ports/geckolib/tools/regen_style_structs.sh

```diff
@@ -65,6 +65,7 @@ fi
   -no-type-renaming                                                 \
   -DMOZILLA_INTERNAL_API                                            \
   -DMOZ_STYLO_BINDINGS=1                                            \
+  -DJS_DEBUG=1                                                      \
```

Comment:
Yeah, if you do a debug build this is probably covered in `mozilla-config.h`. I could try removing this and the `DDEBUG` line and generating multiple build types *might* just work out of the box.

I'll take a look at `mozilla-config.h` to see what is defined depending on the build type, but meanwhile this is needed if you don't make a debug build it seems.